### DISCUSSION TITLE
✅ 🚀 Feature Request: Responsive mode: extra panel option "Show and hide auto mode"

### DISCUSSION THREAD

**Discussion by submitaudio** [elementor/elementor#14021] [Feature Request]
When designing a site with different responsive modes it can become a mess for the clients to edit the content. I need to explain to use the Navigator to show/hide the layers. To be clear please enable this option:

**Option 1 (In the navigator)**
Save view layers in the panel settings and create folders for grouping the devices

**Option 2 (responsive mode panel)**
Create a option to show/hide only the responsive content per device (Mobile, tablet and desktop), Why i want to see the desktop content inside the mobile edit? It's much more work to hide/show content in the navigator each time i reload the page.

**Comment by robertgr52** [comment]
I never noticed but the responsive toggle at the top of the page is really nice and quick to move between modes.

**Comment by nicholaszein** [comment]
Hello @studio67,

the option to toggle between responsive modes is already available at the top of the editor if you enable the new Editor Top Bar experiment in Elementor > Settings > Features that was released with Elementor version 3.12.0. It makes it easy to toggle between modes this way. You should check it out! 😉 

https://elementor.com/help/top-bar-ui/

As for the option to show/hide elements in responsive modes, the option can be found in the User Preferences panel, inside the editor since version 3.3.0.

To access this option using the old editor interface follow the video below:


https://github.com/elementor/elementor/assets/113993460/678c7712-3893-4ae0-80fb-b7239f2d5648



To access this option using the new Top Bar follow the video below:


https://github.com/elementor/elementor/assets/113993460/3352d647-f4ca-4cfd-9180-728440571bdc

----

I hope this helps.

### ✅ I'm closing this request as Solved. 

Feel free to submit new [🚀 Feature Requests](https://github.com/orgs/elementor/discussions/new?category=feature-request), and we will consider it in the future if there is enough demand (votes) from other users.
